### ISSUE TITLE
facet query on cadd fails (unittest MyVariantTest.test_query_facets)

### ISSUE THREAD

**Issue by sirloon** [biothings/myvariant.info#59]
http://myvariant.info/v1/query?q=cadd.gene.gene_id:ENSG00000113368&facets=cadd.polyphen.cat&size=0

gives

```
{
"success": false,
"error": "Could not execute query due to the following exception(s): ['illegal_argument_exception Fielddata is disabled on text fields by default. Set fielddata=true on [cadd.polyphen.cat] in order to load fielddata in memory by uninverting the inverted index. Note that this can however use significant memory. Alternatively use a keyword field instead.']"
}
```

Need to update CADD mapping (and rebuilt pre-merge/cold collection)

**Comment by kevinxin90** [comment]
Follow up on this issue, not sure if it's only related to CADD mapping. I tried to do facets query on other fields, also not working: e.g. http://myvariant.info/v1/query?q=dbsnp.chrom:13&facets=dbsnp.vartype&size=0

error message:
`"Could not execute query due to the following exception(s): ['illegal_argument_exception Fielddata is disabled on text fields by default. Set fielddata=true on [dbsnp.vartype] in order to load fielddata in memory by uninverting the inverted index. Note that this can however use significant memory. Alternatively use a keyword field instead.']"`

**Comment by newgene** [comment]
Fields able to do faceting need to be indexed as "keyword" field, not "text" field. ES disables facets on "text" by default.

**Comment by sirloon** [comment]
fixed in 8e4b67bb435ffcd8e3923adff2fb688161f229b3

**Comment by sirloon** [comment]
more fix 5b479cc44463b45acf649709f77b9156edbdf7ab
